### PR TITLE
Add new parameter enableSeek and OnSeekMediaClockTimer notification

### PR DESCRIPTION
**Pull request by indlu**
SetMediaClockTimer: SDL must support new parameter "enableSeek" and transfer OnSeekMediaClockTimer notification from HMI to mobile app.

 Related task APPLINK-23391

### CODE REVIEW

**Comment by indlu** [comment]
Please review @LevchenkoS, @dcherniev, @okozlovlux , @LuxoftAKutsan

**Comment by indlu** [comment]
Rebuild required

**Comment by ovvasyliev** [comment]
Rebuild required

**Comment by indlu** [comment]
@LuxoftAKutsan, done in 0753d3d

File: src/components/application_manager/test/commands/mobile/simple_notification_commands_test.cc [hunk 1]
```diff
@@ -68,6 +69,7 @@ typedef Types<commands::OnAppInterfaceUnregisteredNotification,
               commands::OnAudioPassThruNotification,
               commands::OnLanguageChangeNotification,
               commands::OnPermissionsChangeNotification,
+              commands::mobile::OnSeekMediaClockTimerNotification,
```
Comment (dcherniev) [inline src/components/application_manager/test/commands/mobile/simple_notification_commands_test.cc]: @indlu I think that changes in this file should be placed in commit related to unit tests.

Comment (indlu) [inline src/components/application_manager/test/commands/mobile/simple_notification_commands_test.cc]: @dchernievyes, you're right, but after review i'm going to squash my commits anyway

Comment (dcherniev) [inline src/components/application_manager/test/commands/mobile/simple_notification_commands_test.cc]: @indlu Well if I am correct, squashing is necessary for commits that are supplementing some already existing commit. As for the unit tests they are related to separate task, and therefore should have their own separate commit.

Comment (indlu) [inline src/components/application_manager/test/commands/mobile/simple_notification_commands_test.cc]: @dcherniev , fixed in 47304f0

**Comment by indlu** [comment]
Rebuild required

**Comment by indlu** [comment]
Rebuild required

**Comment by indlu** [comment]
Rebuild required

**Comment by indlu** [comment]
Rebuild required

**Comment by ovvasyliev** [comment]
Rebuild required

**Pull request merged**